htdocs/core/class/commonobject.class.php

```diff
@@ -1061,18 +1061,25 @@ public function liste_contact($statut = -1, $source = 'external', $list = 0, $co
 	{
         // phpcs:enable
 		global $langs;
+        
+        $langs->load("dict");
 
 		$tab=array();
 
 		$sql = "SELECT ec.rowid, ec.statut as statuslink, ec.fk_socpeople as id, ec.fk_c_type_contact";    // This field contains id of llx_socpeople or id of llx_user
 		if ($source == 'internal') $sql.=", '-1' as socid, t.statut as statuscontact, t.login, t.photo";
 		if ($source == 'external' || $source == 'thirdparty') $sql.=", t.fk_soc as socid, t.statut as statuscontact";
-		$sql.= ", t.civility as civility, t.lastname as lastname, t.firstname, t.email";
+		$sql.= ", t.civility as civility, t.lastname as lastname, t.firstname, t.address, t.zip, t.town, t.fk_pays as country_id, t.fk_departement as state_id";
```

Comment:
```suggestion
		$sql.= ", t.civility as civility, t.lastname as lastname, t.firstname, t.address, t.zip, t.town, t.fk_country as country_id, t.fk_departement as state_id";
```

Maybe it is failing because of this ?